spec/api/catalog_api_spec.rb

```diff
@@ -9,22 +9,255 @@
 
 require 'spec_helper'
 require 'json'
+require 'securerandom'
 
 # Unit tests for SquareConnect::CatalogApi
 # Automatically generated by swagger-codegen (github.com/swagger-api/swagger-codegen)
 # Please update as you see appropriate
 describe 'CatalogApi' do
+
+  beverages = {
```

Comment:
It seems that this doesn't allow the value to be references across different `lets`.